Dockerfile

```
@@ -20,7 +20,18 @@ RUN dotnet publish src/Ombor.API/Ombor.API.csproj \
 
 FROM mcr.microsoft.com/dotnet/aspnet:8.0 AS runtime
 WORKDIR /app
+
+# curl is not in the aspnet base image; it is the probe client for the HEALTHCHECK below.
+RUN apt-get update \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)